agents/python/opamp/http_client.py

```diff
@@ -50,14 +58,41 @@ def run(self):
         
         self.worker()
         
+    def send_first_message_disconnected(self) -> None:
+        first_disconnect_message = opamp_pb2.AgentToServer()
+        
+        agent_description = self.get_agent_description()
+        
+        first_disconnect_message.agent_description.CopyFrom(agent_description)
+        
+        agent_disconnect = self.get_agent_disconnect()
+        first_disconnect_message.agent_disconnect.CopyFrom(agent_disconnect)
+    
+        python_version = f'{sys.version_info.major}.{sys.version_info.minor}.{sys.version_info.micro}'
+        agent_health = self.get_agent_health(component_health=False, last_error=f"Opentelemetry SDK require Python in version 3.8 or higher, [{python_version}] not supported",
+                                       status="UnsupportedRuntimeVersion")
+        first_disconnect_message.health.CopyFrom(agent_health)
+        
+        self.send_agent_to_server_message(first_disconnect_message)
+        
     def send_first_message_with_retry(self) -> None:
         max_retries = 5
         delay = 2
         for attempt in range(1, max_retries + 1):
             try:
-                first_message_server_to_agent = self.send_full_state()
+                # Send first message to OpAMP server, Health is false as the component is not initialized
+                agent_health = self.get_agent_health(component_health=False, last_error="Component not initialized", status="Starting")
```

Comment:
Enum of all the status will avoid mistakes and help understand the different types